cmd/start-rpc-servers/main.go

```diff
@@ -120,7 +120,24 @@ func main() {
 			}
 
 			hostUI := cCtx.Bool(HOST_UI)
-			for _, p := range participants {
+
+			for _, p := range []participant{ivan} {
```

Comment:
Since there is only a single element in this `participant` array, could we remove this `for` loop?